### PR TITLE
Add 'smokesignal' to conda-forge

### PR DESCRIPTION
**Pull request by sodre**

### CODE REVIEW

File: recipes/smokesignal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - smokesignal
+
+about:
+  home: http://github.com/shaunduncan/smokesignal
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (CJ-Wright) [inline recipes/smokesignal/meta.yaml]: This is most likely not correct.

File: recipes/smokesignal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (CJ-Wright) [inline recipes/smokesignal/meta.yaml]: Please remove the comments.

File: recipes/smokesignal/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (CJ-Wright) [inline recipes/smokesignal/meta.yaml]: Could this be a noarch pacakge?

Comment (sodre) [reply recipes/smokesignal/meta.yaml]: My org is having issues including noarch packages that is complicated to describe here. Thus, if I build it noarch we won't be able to use the package on our side.

I hope it is still okay to build it for each architecture separately.

Comment (CJ-Wright) [reply recipes/smokesignal/meta.yaml]: Its fine, we'll just have to wait for Travis then.

Comment (CJ-Wright) [reply recipes/smokesignal/meta.yaml]: Out of curiosity what were the issues you were having with noarch?

Comment (sodre) [reply recipes/smokesignal/meta.yaml]: Hi,
It looks like my build failed for no reasons related to my package. I will skip the OSX build for now, and add it back when the package is fully integrated.

**Comment by sodre** [comment]
@CJ-Wright 
   I switched the package to noarch so we can get it merged into the master branch faster. Are you able to take a look at it soon? 

Thanks!

**Comment by CJ-Wright** [comment]
This is a noarch package and thus only needs to build on circle to pass.

**Pull request merged**